### PR TITLE
chore: Add transition for useTooltip

### PR DESCRIPTION
**Pull request by Chef-Yogi**
demo
demo

### CODE REVIEW

File: packages/uikit/src/hooks/useTooltip/useTooltip.tsx [hunk 1]
```diff
@@ -192,9 +212,15 @@ const useTooltip = (content: React.ReactNode, options: TooltipOptions): TooltipR
...
+  const AnimatedTooltip = (
+    <LazyMotion features={domAnimation}>
+      <AnimatePresence>{visible && tooltip}</AnimatePresence>
+    </LazyMotion>
+  );
+
   return {
     targetRef: setTargetElement,
-    tooltip: tooltipInPortal ?? tooltip,
+    tooltip: tooltipInPortal && AnimatedTooltip,
```
Comment (memoyil) [inline packages/uikit/src/hooks/useTooltip/useTooltip.tsx:223]: Shouldn't we keep the ?? here?

File: packages/uikit/src/hooks/useTooltip/StyledTooltip.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import styled from "styled-components";
+import { m } from "framer-motion";
```
Comment (memoyil) [inline packages/uikit/src/hooks/useTooltip/StyledTooltip.tsx:2]: Can we rename to frameMotion or something for readability

**Comment by chef-jojo** [comment]
What's going on 😅

https://user-images.githubusercontent.com/94336009/159446300-c3bd3f3b-73b3-4a17-ba27-5637d89cb29b.mp4

File: packages/uikit/src/hooks/useTooltip/StyledTooltip.tsx (hunk 2 above)
Comment (Chef-Yogi) [reply packages/uikit/src/hooks/useTooltip/StyledTooltip.tsx:2]: ok

File: packages/uikit/src/hooks/useTooltip/useTooltip.tsx (hunk 1 above)
Comment (Chef-Yogi) [reply packages/uikit/src/hooks/useTooltip/useTooltip.tsx:223]: Yes, should keep

**Comment by Chef-Yogi** [comment]
@chef-jojo @memoyil  
thanks for your reply and reminder, fixed!

**Review by chef-jojo — APPROVED** [review]
Looking good!

**Pull request merged**